NodeStepTemplate.yml

```diff
@@ -48,7 +48,7 @@ steps:
 - checkout: SdkGenerator
   clean: true
   path: s/SdkGenerator
-- checkout: self
+- checkout: ${{ parameters.SelfTemplateResource }}
```

Comment:
Yep. I missed it, but this is exactly the kind of reason why a BuildAll would fail if this weren't set properly.